CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 
 - Bump symbolic to support versioning of CFI Caches. ([#467](https://github.com/getsentry/symbolicator/pull/467))
 - Update symbolic to write versioned CFI Caches, and fix SymCache conversion related to inline-parent offset. ([#470](https://github.com/getsentry/symbolicator/pull/470))
+- Update symbolic to allow processing PDB files with broken inlinee records. ([#477](https://github.com/getsentry/symbolicator/pull/477))
```

Comment:
I may be misunderstanding something, but that is still an open PR, right? How can this update incorporate it?

Reply:
Skipping the pdb was merged actually 8 days ago, it took some more investigation and clearing up some comments that landed an hour ago.